templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>We should update the motivation for this section to be analyzing longitudinal controls, not batch effects. </p><p> </p><p>i.e. if the user has the same control sample in multiple runs, this can be used to analyze how that control shifts over the different runs. The description here, and above, can then be adjusted to emphasize that we're going to be looking at how a single control tissue changes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>